browser/ipfs/ipfs_tab_helper.cc

```diff
@@ -523,6 +532,9 @@ void IPFSTabHelper::DidFinishNavigation(content::NavigationHandle* handle) {
       block_redirect_nav_data && block_redirect_nav_data->enable_redirect_block;
 
   MaybeCheckDNSLinkRecord(handle->GetResponseHeaders(), auto_redirect_blocked);
+  if(IsRedirectToSubLevelUrl(handle, GetCurrentPageURL())) {
```

Comment:
Why it doesn't work as is? 
UpdateLocationBar is also called to update IPFS button state in the omnibar and i see that in the video there is an IPFS button after redirect to /wiki/. Was UpdateLocationBar called?

Reply:
> Was UpdateLocationBar called?
No, It wasn't called because after redirect to /wiki/ the host is the same, and only path changed.
For now we have only this condition: `resolver_->host() != current.host() `, which prevents us from `UpdateLocationBar()` calling.
I propose add the additional one: `(resolver_->host() == current.host() && ipfs_resolved_url_.path() != current.path())` to cover the situation when we have only URL's path changed after redirect